app/views/admin/AdministerUserModal.coffee

```diff
@@ -274,11 +274,17 @@ module.exports = class AdministerUserModal extends ModalView
       }
     }).then (res) =>
       @userSaveState = 'saved'
+      permissions = @user.get('permissions')
+
+      if checked and res.schoolAdministrator == 0
```

Comment:
What does `res.schoolAdministrator == 0` or `res.schoolAdministrator == 1` indicate here

Reply:
@codeluggage want to take this one?

Reply:
Yes - was a slight caching issue on the server side that I was handling here. Basically it would always be one update behind because the model was not updated before reading from it in the response sent back to the client. 

There is a task here to follow up on this to clean up this front end code after ship: https://app.asana.com/0/1115428071773315/1119293250373389/f